test/DistributionTest.js

```diff
@@ -0,0 +1,151 @@
+const DistributionContract = artifacts.require("./DistributionContract");
+const MyTokenn = artifacts.require("./MyTokenn");
+
+const {
+    ether,           // Big Number support
+    constants,    // Common constants, like the zero address and largest integers
+    expectEvent,  // Assertions for emitted events
+    expectRevert, // Assertions for transactions that should fail
+} = require('@openzeppelin/test-helpers');
+
+
+
+const chai = require("./setupchai.js");
+const BN = web3.utils.BN;
+const expect = chai.expect;
+
+contract("Distribution contract", async ([owner, acc2, acc3, acc4]) => {
+
+    let instanceToken;
+    let instanceDistribution;
+
+    before(async () => {
+        instanceToken = await MyTokenn.deployed();
+        instanceDistribution = await DistributionContract.deployed();
+    });
+
+    it("Mint tokens", async () => {
+        expect(await instanceToken.balanceOf(owner)).to.be.bignumber.equal(ether('1000'));
+    });
+
+    it("Transfer deposit tokens", async () => {
+        await instanceToken.approve(DistributionContract.address, ether('400'));
+        await instanceDistribution.deposit(ether('400'));
+        expect(await instanceToken.balanceOf(DistributionContract.address)).to.be.bignumber.equal(ether('400'));
+        expect(await instanceToken.balanceOf(owner)).to.be.bignumber.equal(ether('600'));
+    });
+
+    describe("Add beneficiaries function , fill array", async () => {
+
+        it("Add beneficiaries - fail , address = 0", async () => {
+            let addr = constants.ZERO_ADDRESS;
+            let beneficiaries = [addr, acc3];
+            let amounts = [ether('10'), ether('110')];
+            await expectRevert(instanceDistribution.addBeneficiaries(beneficiaries, amounts), "beneficiary or amount equal to 0");
+        });
+
+        it("Add beneficiaries - fail , caller is not owner", async () => {
+            let beneficiaries = [acc2, acc3];
+            let amounts = [ether('10'), ether('110')];
+            await expectRevert(instanceDistribution.addBeneficiaries(beneficiaries, amounts, { from: acc2 }), "caller is not the owner");
+        });
+
+        it("Add beneficiaries - Array length error", async () => {
+            let beneficiaries = [acc2, acc3];
+            let amounts = [ether('10'), ether('110'), ether('12')];
+            await expectRevert(instanceDistribution.addBeneficiaries(beneficiaries, amounts), "arrays have different length");
+
+        });
+
+        it("Add beneficiaries - success", async () => {
+            let beneficiaries = [acc2, acc3];
+            let amounts = [ether('10'), ether('110')];
+            let _beneficiaries = await instanceDistribution.addBeneficiaries(beneficiaries, amounts);
```

Comment:
_beneficiaries incorrect name to variable, instanceDistribution.addBeneficiaries return you transaction info. just rename variable to `transaction` or `tx`

Reply:
you also should check balanceOfBeneficiares for new beneficiares before and after main action in tests